### ISSUE TITLE
when you releae your code? Is it reproducible?

### ISSUE THREAD

**Issue by vietvo89** [AI4Good24/PsySafe#3]
I tried to follow your paper but it did not work in any case. The setting described in the paper is vague and hard to follow and reproduce your results. No way to benchmark with other research or develop new stuff based on your work.

**Comment by zhangzaibin** [comment]
Sorry for the delay. I am very busy recently. Our main experiments were conducted using the GPT-3.5-0613 model, so the prompt in our paper may not work effectively on the latest models like GPT-4 Turbo or GPT-4o. I recommend combining this dark personality injection technique with some SOTA jailbreak methods to potentially bypass restrictions on advanced LLMs. I plan to open-source the code in this month.

**Comment by zhangzaibin** [comment]
I tested an example in o1-preview with a query on how to create a website that attracts others. The result showed that the dark injection leads to highly dangerous responses. I think LLMs restrict dangerous questions or keywords (bomb, virus...) but do not fully address underlying dark tendencies.

**Comment by zhuyf66** [comment]
Excellent work! Looking forward to your open-source code,  I would appreciate it if you open-source your code as soon as possible!
